Explorer/Assets/DCL/Navmap/NavmapZoomController.cs

```diff
@@ -55,6 +56,9 @@ public void Dispose()
             cts.SafeCancelAndDispose();
         }
 
+        public void SetBlockZoom(bool value) =>

```

Comment:
Any reason this is preferred over a simple getter?

Reply:
there's no need for a public/private getter since the `blockZoom` variable is only used internally. The only need is to set its value externally (from the `FloatingPanelController`)